frontend/src/index.pug

```diff
@@ -109,8 +109,8 @@ html
               label sort groups by
             .mui-select.sort-within-group
               select(
-                  v-model="sortWithinGroupSelection",
-                  v-bind:disabled="filterGroupSelection === 'groupByNone' || isMergeGroup"
+                v-model="sortWithinGroupSelection",
```

Comment:
The spacing changes aren't necessary.

Reply:
Actually, as the standard discussed in #987, the change here is fine.

Reply:
Can we have a PR that fix all the spacing in pug? Otherwise very troublesome to use auto-indent when writing code.

Reply:
Yes, #987 is targeting the issue. Just be patient for it to be merged.